src/sonic-yang-models/yang-models/sonic-types.yang

```diff
@@ -0,0 +1,259 @@
+module sonic-types {
```

Comment:
Please remove this file. It is mistakenly added here and it is part of yang-templates